about.html

```diff
@@ -0,0 +1,96 @@
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,200;0,400;1,600&display=swap" rel="stylesheet">
+    <title>About</title>
+</head>
+<body>
+    <div class="full m-0 p-0">
+        <div class="mobile-menu container-fluid pt-5 gap-5 bg-dark">
+            <ul class="menu gap-1 fs-5 p-5">
+                <li><a href="./index.html">Home</a></li>
+                <li><a href="./about.html">About</a></li>
+                <li><a href="#main-program">Program</a></li>
+                <li><a href="#main-program">Join</a></li>
+                <li><a href="#partner">Sponsor</a></li>
+                <li><a href="">News</a></li>
+                <li><a href="./about.html">Contact</a></li>
+            </ul>
+        </div>
+        <div class="desk-header-menu justify-content-around">
+            <a href="./index.html"><img class="header-logo" src="./image/headerLogo.png" alt="logo"></a>
+            <div class="desktop-menu d-flex">
+                <div class="menu-desktop d-flex gap-5 p-4">
+                    <a href="./index.html">Home</a>
+                    <a href="./about.html">About</a>
+                    <a href="#main-program">Program</a>
+                    <a href="#main-program">Join</a>
+                    <a href="#partner">Sponsor</a>
+                    <a href="">News</a>
+                    <a href="./about.html">Contact</a>
+                </div>
+            </div>
+        </div>
+        <div class="top container-fluid pt-4 p-4">
+            <header class="header fixed-top m-4">
+                <div class="bar" id="bar1"></div>
+                <div class="bar" id="bar2"></div>
+                <div class="bar" id="bar3"></div>
+            </header>
+            <main class="main">
+                <div class="main-content pt-5">
+                    <h2>"Hello Sharing World"</h2>
+                    <H1>CARBON CAPTURE SUMMIT - 2022</H1>
+                </div>
+                <p class="border border-3 border-white mt-5">Carbon capture, utilisation, and storage, or CCUS, is an emerging technology for reducing carbon emissions produced from energy production, heavy industries, and transportation. The captured CO2 can be used as a valuable resource or permanently stored deep underground in geological formations, providing the foundation for carbon removal from the atmosphere. </p>
+                <div class="contact text-center">
+                    <h5>Please contact us via email for any further question</h5>
+                    <h5>About CC Global Summit 2022!</h5>
+                    <h5 id="email" class="email text-dark fs-6"><u>ccsummit2022@ccglobal.org</u></h5>
+                </div>
+            </main>
```

Comment:
- [ ] As specified in the home page, please use `section` tags for the wrappers of the different sections of your projects.

- [ ] Kindly ensure that your text is aligned center in the About page headline heading and description
> Your Design
![image](https://user-images.githubusercontent.com/90556221/183045772-f18f8940-f1b6-4d89-8e58-aa22f84a964a.png)

> Behance Design
![image](https://user-images.githubusercontent.com/90556221/183046428-ea3e135f-e70b-4dea-b139-ce5559b9365d.png)

- [ ] Additionally, please also ensure you have the correct padding and spacings on the left and right like in the Behance design

Reply:
Changed as advised.